File: 프로그래머스/lv0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.java
import java.util.Arrays;

class Solution {
    public String solution(String my_string) {
        String answer = "";
		String to_arr = my_string.toLowerCase();
        // my_string을 모두 소문자로 바꾼다
        
		String[] arr = to_arr.split("");
		Arrays.sort(arr);
        // my_string을 배열로 변환하고 정렬한다
        
		for(int i = 0; i < arr.length; i++) {
			answer += arr[i];
		} // answer에 arr을 순서대로 넣어 문자열로 바꾼다
        return answer;
    }
}

File: 프로그래머스/0/120911. 문자열 정렬하기 （2）/문자열 정렬하기 （2）.js
function solution(myString) {
    
    const lowerCaseString = myString.toLowerCase();
    // 문자열을 모두 소문자로 변환
    
    const sortedArray = lowerCaseString.split('').sort();
    // 문자열을 배열로 변환한 후 정렬
    
    const answer = sortedArray.join('');
    // 배열을 다시 문자열로 합치기
    return answer;
}
